model.py

```diff
@@ -6227,6 +6238,17 @@ class RightsStatus(Base):
     licensepooldeliverymechanisms = relationship("LicensePoolDeliveryMechanism", backref="rights_status")
 
     @classmethod
+    def lookup(cls, _db, uri):
+        if not uri in cls.NAMES.keys():
+            uri = cls.UNKNOWN
+        status, ignore = get_one_or_create(
+            _db, RightsStatus, uri=uri
+        )
+        name = cls.NAMES.get(uri)
+        status.name = name
```

Comment:
Not sure where I should be setting the name. It could just be in the migration, but this will work for a new db as well.